libuavcan/include/uavcan/transport/can_acceptance_filter_configurator.hpp

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (C) 2014 Pavel Kirienko <pavel.kirienko@gmail.com>,
+ *                    Ilia  Sheremet <illia.sheremet@gmail.com>
+ */
+
+#ifndef CAN_ACCEPTANCE_FILTER_CONFIGURATOR_HPP_INCLUDED
+#define CAN_ACCEPTANCE_FILTER_CONFIGURATOR_HPP_INCLUDED
+
+#include <cassert>
+#include <uavcan/data_type.hpp>
+#include <uavcan/error.hpp>
+#include <uavcan/transport/dispatcher.hpp>
+#include <uavcan/node/abstract_node.hpp>
+#include <uavcan/build_config.hpp>
+#include <uavcan/util/multiset.hpp>
+
+namespace uavcan
+{
+
+class CanAcceptanceFilterConfigurator
```

Comment:
A brief comment describing what it is and how to use it would help here.

See an example here: https://github.com/UAVCAN/libuavcan/blob/master/libuavcan/include/uavcan/protocol/node_info_retriever.hpp#L61-L90